core/src/main/java/hudson/model/StringParameterDefinition.java

```diff
@@ -69,7 +70,7 @@ public StringParameterValue getDefaultParameterValue() {
         return v;
     }
 
-    @Extension
+    @Extension @Symbol({"string","stringParam"})
```

Comment:
Ah, so there are classes with `DataBoundConstructor`s that are not `Describable`. I missed that. This needs some more thoughts. So far `@Symbol` was paired up with `@Extension` (which means it's on `Descriptor` and not on `Describable`).

I wonder if maybe we should turn `ParameterValue` into a `Describable`. Or are there others like this?